src/Tizen.NUI/src/public/Window/WindowEvent.cs

```diff
@@ -697,6 +697,12 @@ internal void DisconnectNativeSignals()
                 auxiliaryMessageEventHandler = null;
                 auxiliaryMessageEventCallback = null;
             }
+
+            if (AccessibilityHighlightEventCallback != null)
+            {
+                AccessibilityHighlightEventSignal?.Disconnect(AccessibilityHighlightEventCallback);
```

Comment:
In AccessibilityHighlight.remove, AccessibilityHighlightEventSignal.Empty() is checked.
Here does not check AccessibilityHighlightEventSignal.Empty().
Is it ok?

Reply:
It is ok with following lines in ```AccessibilityHighlight.remove```
```
if (AccessibilityHighlightEventSignal.Empty() == false)
{
  AccessibilityHighlightEventSignal.Disconnect(AccessibilityHighlightEventCallback);
  /* at this point, AccessibilityHighlightEventSignal.Empty() it TRUE */
  AccessibilityHighlightEventSignal.Disconnect(AccessibilityHighlightEventCallback);
}
```
But I just updated the  ```AccessibilityHighlight.remove``` as others do.
```
@@ -1370,6 +1370,10 @@ namespace Tizen.NUI
                         if (AccessibilityHighlightEventSignal.Empty() == false)
                         {
                             AccessibilityHighlightEventSignal.Disconnect(AccessibilityHighlightEventCallback);
+                            if (AccessibilityHighlightEventSignal.Empty() == true)
+                            {
+                                AccessibilityHighlightEventCallback = null;
+                            }
                         }
                     }
                 }
```

Reply:
Thank you for the explanation! :)